.github/workflows/releasecandidate.yml

```diff
@@ -68,9 +68,34 @@ jobs:
           channel: '#team-devops'
           name: 'Workflow Status'
 
+  extract-rc-version:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    outputs:
+      version: ${{ steps.extract-rc-version.outputs.version }}
+      rc_number: ${{ steps.extract-rc-version.outputs.rc_number }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Extract version and RC number
+        id: extract-rc-version
+        run: |
+          TAG_NAME=${GITHUB_REF#refs/tags/}
+          if [[ $TAG_NAME =~ v([0-9]+\.[0-9]+\.[0-9]+)-rc([0-9]+) ]]; then
+            VERSION="${BASH_REMATCH[1]}"
+            RC_NUMBER="${BASH_REMATCH[2]}"
```

Comment:
[q] I'm assuming `BASH_REMATCH` grabs the two capture groups; where does that command come from?

Reply:
[yup that is exactly what it does](https://www.bashsupport.com/bash/variables/bash_rematch/). Just something I looked up